inventory/sample/group_vars/k8s_cluster/k8s-cluster.yml

```diff
@@ -21,7 +21,7 @@ kube_version: v1.31.3
 
 # Where the binaries will be downloaded.
 # Note: ensure that you've enough disk space (about 1G)
-local_release_dir: "/tmp/releases"
+local_release_dir: "/tmp/releases-{{ kube_version }}"
 # Random shifts for retrying failed ops like pushing/downloading
```

Comment:
Is there a specific reason for this ? Does not seem needed

Reply:
I have installed k8s with different version many times which the local release dir stored so many binary files of many version. Just only applies to my scene, I had 
 rollback this change.